.vsts-ci.yaml

```diff
@@ -0,0 +1,9 @@
+# trigger:
```

Comment:
This shouldn't show up as a code change it was already merged it looks like the rebase isn't clean